dev-util/emscripten-fastcomp/emscripten-fastcomp-1.37.9.ebuild

```diff
@@ -40,6 +40,8 @@ src_configure() {
 	local mycmakeargs=(
 		# avoid clashes with sys-devel/llvm
 		-DCMAKE_INSTALL_PREFIX="/usr/share/${P}"
+		-DLLVM_OCAML_INSTALL_PATH="/usr/share/${P}/ocaml"
```

Comment:
cmake-utils.eclass already adds a prefix in https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/cmake-utils.eclass#n812, see also https://cmake.org/cmake/help/v3.0/variable/CMAKE_INSTALL_PREFIX.html for more details.